### PR TITLE
[MX-263] Removes individual artist artwork rails from home

### PR DESCRIPTION
**Pull request by ashfurrow**
Every now and then on the home page, we would include a rail for artworks by either an artist you follow, or a recommended artist based on who you already follow. It's not really clear to the user why they see this content, and the content is duplicated in the "works for you" screen, so let's remove it from home.

Here's a side-by-side comparison as you scroll down home. You can see the the `Keith Haring` and `Jen Wink Hays` artwork rails are gone:

<details><summary>screenshots</summary>

| Before | After |
|---|---|
| OldHome1  | NewHome1  |
| OldHome2  | NewHome2  |
|  OldHome3 |  NewHome3 |
| OldHome4   | NewHome4 |
| OldHome5 |  |

</details>

Note: I'm leaving in [some logic](https://github.com/artsy/eigen/blob/4f8ec8b0b948cbdfda8cda8ffa6819faf576bde1/src/lib/Scenes/Home/Components/ArtworkRail.tsx#L41-L42) to handle artwork rails subtitles, in case we add these back in later.

### CODE REVIEW

**Comment by ashfurrow** [comment]
Waiting for #3223 to get merged so I can add a changelog entry.

File: CHANGELOG.yml [hunk 1]
```diff
@@ -11,6 +11,7 @@ upcoming:
     - updates CityGuide BMW logo - ashleyjelks
     - Adds Trending Artist Series Collection Hub Rail - ashleyjelks
     - Fix untappable forgot password bug - brian
+    - Removes unnecessary artist artwork cards from home - ash
```
Comment (pvinis) [inline CHANGELOG.yml:14]: "unnecessary artist", so harsh :p

**Comment by pvinis** [comment]
Let's try this..

**Comment by pvinis** [comment]
#MergeOnGreen

**Comment by pvinis** [comment]
Success.

File: CHANGELOG.yml (hunk 1 above)
Comment (ashfurrow) [reply CHANGELOG.yml:14]: Haha 🙈

**Pull request merged**